src/main/java/com/blibli/future/service/FileReaderComponentImpl.java

```diff
@@ -0,0 +1,87 @@
+package com.blibli.future.service;
+
+import com.blibli.future.service.api.FileReaderComponent;
+import org.apache.poi.hssf.usermodel.HSSFWorkbook;
+import org.apache.poi.ss.usermodel.Row;
+import org.apache.poi.ss.usermodel.Sheet;
+import org.apache.poi.ss.usermodel.Workbook;
+import org.apache.poi.xssf.usermodel.XSSFWorkbook;
+import org.springframework.stereotype.Component;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
```

Comment:
Hmm... That's right. Why did I write it as Component? Thanks for pointing it out, I'll change it.